expat/CMakeLists.txt

```diff
@@ -124,6 +124,12 @@ if(EXPAT_WITH_LIBBSD)
     endif()
 endif()
 
+if(MSVC)
+    if(MSVC_VERSION VERSION_LESS 1800)
+        message(SEND_ERROR "MSVC_VERSION ${MSVC_VERSION} is not a supported Visual Studio compiler version")
```

Comment:
It would be great to communicate here that `1800` is `Visual Studio 12.0/2013` and that that's the minimum requirement, ideally adding a link o https://cmake.org/cmake/help/latest/variable/MSVC_VERSION.html for reference and an in-code comment that those numbers need to be kept in sync with the readme file. What do you think?

Reply:
Communicate as source comment or as message for user?

Reply:
The keep-in-sync should be a code comment but all else can be part of the message to the user as it may provide value. What do you think?

Reply:
OK, done.